tests/test_books.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.mark.parametrize("bookid, title", [
+    ("nneBa6-mWfgC", "Coders at Work"),
+    ("__CvAFrcWY0C", "Unlimited Power"),
+    ("3V_6DwAAQBAJ", "Power Vs. Force"),
+    ("bK1ktwAACAAJ", "177 Mental Toughness Secrets of the World Class"),
+])
+def test_homepage_shows_books(client, books, bookid, title):
+    response = client.get('/')
+    expected = ('<a href="/books/{bookid}"><img class="thumbNail" '
+                'src="http://books.google.com/books?id={bookid}&'
+                'printsec=frontcover&img=1&zoom=1&source=gbs_gdata" '
+                'alt="{title}"></a>').format(bookid=bookid,
+                                             title=title)
+    assert expected in response.content.decode()
+
+
+@pytest.mark.parametrize("snippet, is_in", [
```

Comment:
using parametrize here over 6 asserts in the test, the advantage is that pytest does not stop upon the first failure and you get better pytest error output